packages/web/src/components/ui/PackageVersion/PackageVersion.tsx

```diff
@@ -1,161 +1,70 @@
-import React, { useState } from 'react';
+import React from 'react';
 import styles from './PackageVersion.module.scss';
 import clsx from 'clsx';
 import { Icon } from '../Icon/Icon';
-import CircularProgress from '../CircularProgress/CircularProgress';
-import ProgressBar from '../ProgressBar/ProgressBar';
-import { formatNumber } from 'utils/helpers';
-
-type Module = {
```

Comment:
In this case, i'd rather leave this (and related) code commented-out or, better yet, create an issue with link to PR/commit where it was removed. We'll be needing this in nearest future.